Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1268,11 +1264,7 @@ - (void)processNextQueueItem {
                 [req processResponse:nil error:[NSError errorWithDomain:BNCErrorDomain code:BNCInitError userInfo:@{ NSLocalizedDescriptionKey: @"Branch User Session has not been initialized" }]];
                 return;
             }
-            
-            if (![req isKindOfClass:[BranchCloseRequest class]]) {
-                [self clearTimer];
-            }
-            
+                        
```

Comment:
My perceived experience is that when the app transitions to/from background, the main queue is paused so nothing executes there, but other work queues may fire a callback when starting/stopping.  So things don't run indefinitely in the background and the network queue isn't going to empty in the background.

But honest question:  Is it important that these events strictly happen after the next successful  initSession?